nimsuggest/tests/timport_multi_modules.nim

```diff
@@ -0,0 +1,15 @@
+discard """
+$nimsuggest --tester $file
+>def $1
+def;;skModule;;minclude_types;;*;;$file;;10;;15;;"";;100
+>def $2
+def;;skModule;;minclude_types;;*;;$file;;11;;15;;"";;100
+>def $3
+def;;skModule;;minclude_import;;*;;$file;;13;;10;;"";;100
+"""
+import fixtures/[minclude_import, minclud#[!]#e_types]
+import fixtures/[minclude_types as typ#[!]#es]
```

Comment:
It looks like suggestions for the `import x as y` syntax (without the brackets) also didn't work previously. Please add an additional test for this case here.

In addition, I don't think the 'def' should be on the alias. Rather it should be produced on the original module name (`minclude_types` in this case).

Reply:
well, from my point of view, it should be on both side, if only choose one side, I'll choose the alias side, let's say `import x as y`, then the code `y.Xfield` when hover on `y` should show the module document.

Reply:
> let's say `import x as y`, then the code `y.Xfield` when hover on `y` should show the module document.

Yes, but I'm not talking about usage. On usage, only the alias is available and visible, but in the import statement, the original name is immediately visible.

In addition, `onSymImport`, as the name implies, is called for the symbol of the module (and not the alias) -- the source position (and thus the `def`) should reflect that.

Reply:
you mean the user have to hover on the original module name not the alias? that's weird.

Reply:
I think having to hover over the original module name within the `import` statement to show the module's document comment makes sense, yes.

In any way, given that the whole importing logic needs to be reworked (probably from the ground up) anyway, I think it can stay like it is for now. However, please document the behaviour and surrounding uncertainty in the PR body.